shared/chat/conversations-list/container.js

```diff
@@ -13,7 +16,54 @@ class ConversationListContainer extends Component {
   }
 
   render () {
-    return <ConversationList {...this.props} />
+    const rows = this.props.inbox.map(conversation => {
```

Comment:
We hoist up this re-used logic so it can be shared in the platform specific components. They can be 'dumber' and know less about the actual structure of the data in the store